project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -80,14 +83,14 @@ letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
 A request is approved when all of the following criteria are met:
 
 * The pull request has been open for at least 72 hours.
-* The request has approval from at least four members of @openjs-foundation/cpc.
-* No members of @openjs-foundation/cpc have objected to the request.
+* The request has approval from at least four members of @openjs*foundation/cpc.
+* No members of @openjs*foundation/cpc have objected to the request.
```

Comment:
```suggestion
* The request has approval from at least four members of @openjs-foundation/cpc.
* No members of @openjs-foundation/cpc have objected to the request.
```

Reply:
I have already fixed those in the last commit

Reply:
these are still not fixed.

Reply:
wow i am sure why because locally it shows fine i just pushed it again hoping this is the fix